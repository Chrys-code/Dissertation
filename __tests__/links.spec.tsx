import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import Links from '../pages/links/index';


test('true is truthy', () => {
    expect(true).toBe(true);
});

test('false is falsy', () => {
    expect(false).toBe(false);
});

describe('Index', () => {
    it('should render without throwing an error', () => {
        const testRenderer = TestRenderer.create(<Links />);
        expect(testRenderer.toJSON()).toMatchSnapshot();
    });
});