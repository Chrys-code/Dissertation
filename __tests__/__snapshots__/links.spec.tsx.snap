// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render without throwing an error 1`] = `
Array [
  <div
    className="navbar"
  >
    <div
      className="container"
    >
      <button
        className="btn"
        type="submit"
      >
        ‚Üê
      </button>
      <div
        className="profile"
      >
        <p
          id="profile_name"
        >
          Name here
        </p>
        <button
          className="btn"
          type="submit"
        >
          <img
            alt="avatar"
            src="/images/avatar.svg"
          />
        </button>
      </div>
    </div>
  </div>,
  <div
    className="container"
  >
    <div
      className="link"
      href="https://www.w3schools.com/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <div
        className="icon"
      >
        <img
          alt=""
          src="/images/globe.svg"
        />
      </div>
      <div>
        <h2>
          W3 schools
        </h2>
        <p>
          test link description
        </p>
      </div>
    </div>
    <div
      className="link"
      href="https://www.w3schools.com/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <div
        className="icon"
      >
        <img
          alt=""
          src="/images/globe.svg"
        />
      </div>
      <div>
        <h2>
          W3 schools
        </h2>
        <p>
          test link description
        </p>
      </div>
    </div>
    <div
      className="link"
      href="https://www.w3schools.com/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <div
        className="icon"
      >
        <img
          alt=""
          src="/images/globe.svg"
        />
      </div>
      <div>
        <h2>
          W3 schools
        </h2>
        <p>
          test link description
        </p>
      </div>
    </div>
    <div
      className="link"
      href="https://www.w3schools.com/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <div
        className="icon"
      >
        <img
          alt=""
          src="/images/globe.svg"
        />
      </div>
      <div>
        <h2>
          W3 schools
        </h2>
        <p>
          test link description
        </p>
      </div>
    </div>
    <button
      className="btn"
    >
      Add
    </button>
  </div>,
]
`;
